### DISCUSSION TITLE
What is "FreeU" and how does it works?

### DISCUSSION THREAD

**Discussion by Extraltodeus** [Comfy-Org/ComfyUI#1636] [Q&A]
There is that new node in the "_for_testing" menu and while it does gives different results with a same seed I can't put my finger on what it does and how to use it. Does anybody knows anything?

**Comment by Extraltodeus** [comment]
I've found this if anybody is interested:

https://github.com/ChenyangSi/FreeU

**Reply by manuiageek** [reply]
nobody is interested

**Reply by roop03988** [reply]
Thanks for sharing @Extraltodeus ! Helped me understand the params at least

**Comment by Mikerhinos** [comment]
From the tests I made when it got added it helps with the prompt following, but I may be wrong.
